### PR TITLE
add doc pointer on integration tests in contributor guide

### PR DESCRIPTION
**Pull request by dmitris**
PR adds a link to test/README.md / "Aya Integration Tests" section.

### CODE REVIEW

**Comment by dmitris** [comment]
with the panic removed, the output now becomes:
```
Error in integration_test::tests::load::pin_lifecycle: Invalid link
thread 'main' panicked at 'Invalid link', test/integration-test/src/main.rs:72:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Aborted
```

File: test/integration-test/src/main.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ macro_rules! exec_test {
     ($test:expr) => {{
         info!("Running {}", $test.name);
         if let Err(e) = ($test.test_fn)() {
+            println!("Error in {}: {}", $test.name, e);
```
Comment (vadorovsky) [inline test/integration-test/src/main.rs:34]: This is just a bit better than what we have currently, but this still doesn't change the backtrace and doesn't show which particular line of code is causing the error.

I described my idea of fix in this issue: #421

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ Make sure that your work is tracked by an issue or a (draft) pull request, this
 helps us avoid duplicating work. If your work includes publicly visible changes,
 make sure those are properly documented as explained in the section above.
 
+### Running tests
+Run the unit tests with `cargo test`. See [Aya Integration Tests](https://github.com/aya-rs/aya/blob/main/test/README.md) regarding running the integration tests.
```
Comment (vadorovsky) [inline CONTRIBUTING.md:34]:
```suggestion
Run the unit tests with `cargo test`. See [Aya Integration Tests](https://github.com/aya-rs/aya/blob/main/test/README.md) regarding running the integration tests.

```

(add a space before `### Commits`)

File: test/integration-test/src/main.rs (hunk 1 above)
Comment (dmitris) [reply test/integration-test/src/main.rs]: removed, will do in a separate PR

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**